snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedSnapshot.java

```diff
@@ -160,7 +169,7 @@ public int hashCode() {
     int result = getDirectory().hashCode();
     result = 31 * result + checksumFile.hashCode();
     result = 31 * result + (int) (getChecksum() ^ (getChecksum() >>> 32));
-    result = 31 * result + getMetadata().hashCode();
+    result = 31 * result + getSnapshotId().hashCode();
     return result;
```

Comment:
:wrench: :question: Shouldn't metadata be part of the checksum? Unless it's required for backwards compatability somehow?

Reply:
Metadata is already included in the checksum, isn't enough?

Reply:
I see, that makes sense :+1: